healthcare/consent/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nodejs-docs-samples-healthcare",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "repository": "GoogleCloudPlatform/nodejs-docs-samples",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "test": "mocha system-test/*.test.js --timeout=60000"
+  },
+  "devDependencies": {
+    "mocha": "^9.0.0"
+  },
+  "dependencies": {
+    "@googleapis/healthcare": "^3.0.0",
+    "mocha-test": "^2.0.4",
+    "uuid": "^8.0.0"
```

Comment:
Since `uuid` is only used in tests, it should be a dev dependency.

Reply:
Done.